project/AppDependencies.scala

```diff
@@ -0,0 +1,47 @@
+import sbt._
+
+object AppDependencies {
+  import play.core.PlayVersion
+
+  val compile = Seq(
+    "uk.gov.hmrc"       %% "bootstrap-play-26" % "1.4.0",
+    "com.typesafe.play" %% "play-json"         % "2.6.14",
+    "com.amazonaws"     % "aws-java-sdk-s3"    % "1.11.699"
+  )
+
+  trait TestDependencies {
```

Comment:
Is this trait actually useful? It seems like it just adds some indirection, but we could just do 

    def apply() = compile ++ commonTestDependencies("test") ++ commonTestDependencies("it")

Reply:
Have eliminated the trait, and simply specify multiple scopes as required.